.github/workflows/cucumber-integration-test.yaml

```diff
@@ -40,9 +41,16 @@ jobs:
           ADMIN_USER_API_KEY: ${{ secrets.adminUserApiKey }}
           ISSUE_FILTER: ${{ inputs.executionFilter }}
           CUCUMBER_PUBLISH_TOKEN: ${{ secrets.cucumberPublishToken }}
-        # workaround replacement since injecting the token via environment variable does not work
         run: |
-          sed -i "s/CUCUMBER_TOKEN_IRS_PLACEHOLDER/${CUCUMBER_PUBLISH_TOKEN}/g" irs-cucumber-tests/src/test/java/org/eclipse/tractusx/irs/cucumber/RunCucumberTest.java
+          if [ -z "${{ secrets.cucumberPublishToken }}" ]; then
+            echo "Publish cucumber report disabled"
+            sed -i "s/PUBLISH_ENABLED_FLAG/false/g" irs-cucumber-tests/src/test/java/org/eclipse/tractusx/irs/cucumber/RunCucumberTest.java
```

Comment:
This sed replaces the `PUBLISH_ENABLED_FLAG` in RunCucumberTest. There is not such placeholder value in this file